app/src/main/java/com/zeapo/pwdstore/utils/Extensions.kt

```diff
@@ -108,17 +110,11 @@ fun SharedPreferences.getString(key: String): String? = getString(key, null)
 
 suspend fun FragmentActivity.commitChange(
     message: String,
-    finishWithResultOnEnd: Intent? = null,
-    finishActivityOnEnd: Boolean = true,
```

Comment:
We'll need to audit this change and confirm there's no callers left who relied on the implicit finish and haven't been fixed.